dev-libs/pigpio/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>gentoo@k8n.de</email>
```

Comment:
gentoo proxy maintainers project should be listed here too (https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers)

Reply:
Done :-)